cylc/flow/cfgspec/globalcfg.py

```diff
@@ -79,6 +80,17 @@
     from those higher up (but if a setting is present in a file higher up and
     not in any files lower down, it will not be overridden).
 
+    .. note::
+
+       For example, if the following files had these settings:
+
+       .. csv-table::
+          :header: File, Setting A, Setting B, Setting C
+
+          <site-conf-path>/flow/global.cylc, Hello, World, <not set>
+          ~/.cylc/flow/global.cylc, <not set>, Mars, 42
+          **Actual config**, **Hello**, **Mars**, **42**
+
```

Comment:
I don't really understand the example in this format.

Reply:
I think I get the example, but I think it is a tad confusing..
I think you are saying that the actual config will fill from wherever it can. So, it can pull from both config files, yes?

Reply:
I think it's trying to demonstrate that the configs are applied in order and that later configs can override earlier ones.

Reply:
No worries. If it doesn't help anyone else, I'll remove it.